packages/web3-common/src/formatters.ts

```diff
@@ -47,14 +48,6 @@ export const outputProofFormatter = (proof: Proof): Proof => ({
  */
 export const outputBigIntegerFormatter = (number: Numbers) => toNumber(number);
 
-/**
- * Returns true if the given blockNumber is 'latest', 'pending', or 'earliest.
- */
-export const isPredefinedBlockNumber = (blockNumber: string) =>
-	PredefinedBlockNumbers.LATEST === blockNumber ||
-	PredefinedBlockNumbers.PENDING === blockNumber ||
-	PredefinedBlockNumbers.EARLIEST === blockNumber;
-
```

Comment:
Renamed and moved to [web3-utils](https://github.com/ChainSafe/web3.js/blob/c7933b378db2e220ca0444f3481e1d0c4cb16d04/packages/web3-utils/src/validation.ts#L419)